src/fiskaltrust.ifPOS/v1/de/Models/TseInfo.cs

```diff
@@ -6,37 +6,119 @@ namespace fiskaltrust.ifPOS.v1.de
     [DataContract]
     public class TseInfo
     {
+        /// <summary>
+        /// maximal number of clientId's supported
+        /// </summary>
         [DataMember(Order = 10)]
         public long MaxNumberOfClients { get; set; }
 
+        /// <summary>
+        /// current number of clients registered
+        /// </summary>
         [DataMember(Order = 20)]
         public long CurrentNumberOfClients { get; set; }
 
+        /// <summary>
+        /// current list of clientId's registered
+        /// </summary>
         [DataMember(Order = 30)]
         public IEnumerable<string> CurrentClientIds { get; set; }
 
+        /// <summary>
+        /// maximal number of started transactions supported
+        /// </summary>
         [DataMember(Order = 40)]
         public long MaxNumberOfTransactions { get; set; }
 
+        /// <summary>
+        /// current number of started transactions
+        /// </summary>
         [DataMember(Order = 50)]
         public long CurrentNumberOfTransactions { get; set; }
 
-        [DataMember(Order = 60)]
-        public string TsePublicKeyBase64 { get; set; }
+        /// <summary>
+        /// current list of started stransactions
+        /// </summary>
+        [DataMember(Order =60)]
+        public IEnumerable<ulong> CurrentTransactionNumberss { get; set; }
 
-        [DataMember(Order = 70)]
-        public string TseSerialNumberOctet { get; set; }
+        /// <summary>
+        /// maximum number of signatures
+        /// </summary>
+        [DataMember(Order =70)]
+        public long MaxNumberOfSignatures { get; set; }
 
-        [DataMember(Order = 80)]
-        public TseStates CurrentState { get; set; }
+        /// <summary>
+        /// current number of signatures created
+        /// </summary>
+        [DataMember(Order =80)]
+        public long CurrentNumbersOfSignatures { get; set; } //signaturecounter
 
-        [DataMember(Order = 90)]
-        public IEnumerable<string> CertificatesBase64 { get; set; }
+        /// <summary>
+        /// maximum size of log-memory in bytes
+        /// </summary>
+        [DataMember(Order =90)]
+        public long MaxLogMemorySize { get; set; }
 
-        [DataMember(Order = 100)]
-        public IEnumerable<string> SerialNumbersOctet { get; set; }
+        /// <summary>
+        /// current size of used log memory in bytes
+        /// </summary>
+        [DataMember(Order =100)]
+        public long CurrentLogMemorySize { get; set; }
 
+        /// <summary>
+        /// current state in device lifecycle
+        /// </summary>
         [DataMember(Order = 110)]
+        public TseStates CurrentState { get; set; }
+
+        /// <summary>
+        /// identification of the device firmware
+        /// </summary>
+        [DataMember(Order =120)]
```

Comment:
Could you maybe remove the duplicate newlines below and press CTRL+K, D once to format the file and get rid of the wrong spaces (`Order  =120`)?